app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        cmd = command.split()
+        command_name = cmd[0]
+        source_name = cmd[1]
+        copy_name = cmd[2]
+    except Exception:
+        print("Error in command argument!")
+        return None
+
+    if command_name == "cp":
+        if source_name != copy_name:
```

Comment:
You can join two if to one
```suggestion
    if command_name == "cp" and source_name != copy_name:
```